src/templates/finops-hub/modules/storage.bicep

```diff
@@ -140,8 +326,8 @@ resource identityRoleAssignments 'Microsoft.Authorization/roleAssignments@2022-0
   }
 }]
 
-resource uploadSettings 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
-  name: '${storageAccountName}_uploadSettings'
+resource uploadSettings 'Microsoft.Resources/deploymentScripts@2023-08-01' = {
+  name: '${safeHubName}_uploadSettings'
```

Comment:
nit: Shouldn't it still be storageAccountName, since that's where it's uploading to?

Reply:
Applied